common/djangoapps/adaptive_learning/migrations/0002_courseadaptivelearningflag.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import openedx.core.djangoapps.xmodule_django.models
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('adaptive_learning', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CourseAdaptiveLearningFlag',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('change_date', models.DateTimeField(auto_now_add=True, verbose_name='Change date')),
+                ('enabled', models.BooleanField(default=False, verbose_name='Enabled')),
+                ('course_id', openedx.core.djangoapps.xmodule_django.models.CourseKeyField(max_length=255, db_index=True)),
```

Comment:
@tomaszgy It would be nice to reduce this to:

```python
                ('course_id', CourseKeyField(max_length=255, db_index=True)),
```

Reply:
👍